Makefile

```diff
@@ -8,7 +8,7 @@ PLATFORMS ?= linux_amd64 linux_arm64
 
 CODE_GENERATOR_REPO ?= https://github.com/aws-controllers-k8s/code-generator.git
 CODE_GENERATOR_COMMIT ?= f8c0f4711a91e50335450521e3c49673d135d56d
-GENERATED_SERVICES := $(shell find ./apis -type f -name generator-config.yaml | cut -d/ -f 3 | tr '\n' ' ')
+GENERATED_SERVICES ?= $(shell find ./apis -type f -name generator-config.yaml | cut -d/ -f 3 | tr '\n' ' ')
```

Comment:
@iAnomaly  what is the reason for changing this ?

Reply:
Hey @haarchri, I was iterating on just `rds` so I wanted to be able to do `GENERATED_SERVICES=rds make generate` and not wait for all the other services.

If you don't set `GENERATED_SERVICES` the behavior is unchanged/the way it was before.